Makefile

```diff
@@ -45,11 +45,10 @@ ci-non-test: verify licenses vulncheck ## Runs checks other than tests
 run: ## Run Karpenter controller binary against your local cluster
 	SYSTEM_NAMESPACE=${KARPENTER_NAMESPACE} \
 		KUBERNETES_MIN_VERSION="1.19.0-0" \
-		LEADER_ELECT=false \
+		DISABLE_LEADER_ELECT=true \
 		DISABLE_WEBHOOK=true \
 		CLUSTER_NAME=${CLUSTER_NAME} \
 		INTERRUPTION_QUEUE=${CLUSTER_NAME} \
-		FEATURE_GATES="Drift=true" \
```

Comment:
As part of the run, should we enable the feature gate for spotToSpot consolidation? I think that we enable this by default when we deploy the helm chart in the `make apply`

Reply:
We don't do that in apply today, should we?

Reply:
Nm - it's passed as part of helm opts